platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/style/layers/TransitionOptions.java

```diff
@@ -42,6 +42,7 @@ public static TransitionOptions fromTransitionOptions(long duration, long delay)
    *
    * @return the transition duration
    */
+  @Keep
```

Comment:
[diff-7d18a03153f089558a339ae9defc022aR14](https://github.com/mapbox/mapbox-gl-native/pull/13681/files#diff-7d18a03153f089558a339ae9defc022aR14) references a field, not the getter method. I'm not sure that protecting the getter from being obfuscated will protect the field as well. All good if that's the case, but if it's not, could you move the `@Keep` declaration onto the field instead?